framework_tvb/tvb/config/init/introspector_registry.py

```diff
@@ -120,6 +126,16 @@ class IntrospectionRegistry(object):
         CreateAlgorithmCategoryConfig: import_adapters(tvb.adapters.creators, ALL_CREATORS),
     }
 
+    DATATYPE_REMOVERS = {
```

Comment:
How about moving this constant next to `from tvb.adapters.datatypes.db import ALL_DATATYPES` ?